### ISSUE TITLE
Drag & Drop functionality

### ISSUE THREAD

**Issue by ErnestasMitkus** [ZenitechSoftware/planning-chess#105]
[Should Have/Could Have]
Would be nice to be able to drag & drop the chess pieces on the board.

**Comment by Eugenijus** [comment]
Yes, during test use in one of the Zenitech project people intuitively were trying to move chess peaces by drag & drop and expected it to work :)
